aten/src/ATen/native_parse.py

```diff
@@ -99,6 +103,33 @@ def parse_arguments(args, func_decl, func_name, func_return):
     return arguments
 
 
+def parse_return_arguments(return_decl, inplace):
```

Comment:
What changes were made in this block?

Reply:
this is totally new, it just borrows some minimal code from the normal argument parsing.